src/index.js

```diff
@@ -560,7 +560,7 @@ class Offline {
             // https://hapijs.com/api#route-configuration doesn't seem to support selectively parsing
             // so we have to do it ourselves
             const contentTypesThatRequirePayloadParsing = ['application/json', 'application/vnd.api+json'];
-            if (contentTypesThatRequirePayloadParsing.includes(contentType)) {
+            if (contentTypesThatRequirePayloadParsing.includes(contentType) && request.payload && request.payload.length > 1) {
```

Comment:
Can you tell me why the `> 1` ?

Reply:
as far as I know the string that `JSON.parse` takes is at minimum `{}`, anything shorter than that fails :)